### PR TITLE
feat: Support for building with Java 11 (SDKCF-4917)

### PR DESCRIPTION
**Pull request by leila-gu**

### CODE REVIEW

File: versions/versions.gradle [hunk 1]
```diff
@@ -6,8 +6,8 @@
 // than the consumer applications.
 ext.CONFIG.versions = [
     // Java compatibility target version
-    java      : '1.7',
-    kotlin    : '1.4.10',
+    java      : '11',
+    kotlin    : '1.4.31',
```
Comment (vincentdelmundo) [inline versions/versions.gradle:10]: should we already update this to 1.5.x?

**Review by vincentdelmundo — COMMENTED** [review]
Looks good. Just one question.

File: versions/versions.gradle (hunk 1 above)
Comment (leila-gu) [reply versions/versions.gradle:10]: not all projects are updated, SDKUtils is still using v1.3.50

Comment (vincentdelmundo) [reply versions/versions.gradle:10]: yes, I think we can update this
and let other projects to use old one manually if necessary.

Comment (leila-gu) [reply versions/versions.gradle]: Updated to v1.5.21

**Review by vincentdelmundo — APPROVED** [review]

**Pull request merged**